### PR TITLE
DDF-2354 fixed several usability issues with the logout, login, and admin console pages

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?

 - users can return to the previous page
 - users can proceed to the landing page
 - users can proceed to the login page
 - added headers to login and logout
 - fixed issue with iframe not sizing correctly on logout page
 - fixed issue with Admin Console header disappearing
 - fixed issue with logout page trying to get google font

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@shaundmorris @coyotesqrl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by coyotesqrl** [comment]
👍

**Comment by shaundmorris** [comment]
Manually tested 👍 
sign out prompt

**Comment by pklinef** [comment]
Should this be backported to 2.9.x?

File: platform/security/servlet/security-servlet-logout/Gruntfile.js [hunk 1]
```diff
@@ -93,6 +93,14 @@ module.exports = function (grunt) {
                 tasks: ['bower']
             }
         },
+        sed: {
+            imports: {
+                path: 'target/webapp/lib/bootswatch/flatly',
+                pattern: '@import url\\("//fonts.googleapis.com/css\\?family=Lato:400,700,400italic"\\);',
```
Comment (shaundmorris) [inline platform/security/servlet/security-servlet-logout/Gruntfile.js]: @stustison @pklinef was this an issue if DDF was being run offline?

Comment (stustison) [inline platform/security/servlet/security-servlet-logout/Gruntfile.js]: Yes it was.

**Comment by stustison** [comment]
I plan to backport this as soon as it is merged.

File: platform/security/handler/security-handler-guest/src/main/webapp/css/style.css [hunk 2]
```diff
@@ -16,3 +16,43 @@
...
+    color: #95a5a6;
+    white-space: nowrap;
+    vertical-align: middle;
+    height: 51px;
+}
+
+.app-header > label {
+    width: 33%;
+    overflow: hidden;
+    -ms-text-overflow: ellipsis;
```
Comment (pklinef) [inline platform/security/handler/security-handler-guest/src/main/webapp/css/style.css]: IE6 supports text-overflow.  IE8 added the prefixed version but you do not need it.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

**Comment by pklinef** [comment]
Please attach a screenshot showing the full logout page with top banner.

**Comment by stustison** [comment]
screen shot 2016-08-05 at 9 26 28 am

**Comment by pklinef** [comment]
A version of the logo based on the SVG source image.

ddf

**Comment by pklinef** [comment]
👍

**Comment by pklinef** [comment]
This one looks more like the current logo.

ddf

**Pull request merged**